src/utils/radio.ts

```diff
@@ -0,0 +1,2 @@
+export const getRadioStationUrl = (stationName: string) =>
+  `https://w.soundcloud.com/player/?url=${stationName}&amp;start_track=0&amp;single_active=true&amp;show_artwork=false`;
```

Comment:
```suggestion
export const getRadioStationUrl = (stationName: string) =>
  stationName && `https://w.soundcloud.com/player/?url=${stationName}&amp;start_track=0&amp;single_active=true&amp;show_artwork=false`;
```

Reply:
shouldn't cause any issues if we pass an empty string there